crates/rattler_conda_types/src/repo_data/mod.rs

```diff
@@ -40,27 +42,27 @@ pub struct RepoData {
     pub info: Option<ChannelInfo>,
 
     /// The tar.bz2 packages contained in the repodata.json file
-    #[serde(default, serialize_with = "sort_map_alphabetically")]
-    pub packages: FxHashMap<String, PackageRecord>,
+    #[serde(default, serialize_with = "sort_indexmap_alphabetically")]
```

Comment:
```suggestion
    #[serde(default)]
```

We can remove the serialize_with code since the insertion order is what we want.